psi4/src/psi4/libmints/writer.h

```diff
@@ -81,6 +81,22 @@ class PSI_API MoldenWriter {
                std::shared_ptr<Vector> OccB, bool dovirtual);
 };
 
+class PSI_API MoldenWriter {
```

Comment:
I think you misunderstood me earlier. Get rid of this "new" class and move the `PSI_DEPRECATED` warnings to the functions of the _existing_ `MoldenWriter` class. This is probably why the build is failing.